integrations/phone/examples/common-interactions/README.md

```diff
@@ -0,0 +1,61 @@
+## Phone Starter Kit
+
+This example contains Watson Assistant phone integration actions that demonstrate how to:
+
+- How to use the phone and SMS channel for 2-factor authentication via the [send a text message](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-phone-actions#phone-actions-sms.)
+- How to tune the Watson Speech-To-Text parameters in the Greetings action for background noise (`background_audio_suppression`) and increase the speech capture time (`end_of_phrase_silence_time`) by using the [speech_to_text](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-phone-actions#phone-actions-speech-advanced) response type.
+- How to disable and enable Text-to-Speech barge-in via the [text_to_speech](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-phone-actions#phone-actions-text-advanced) response type.
+- How to handle the input timeout from the phone channel via the `vgwPostResponseTimeout` keyword.
+- How to disconnect the call when the caller is done using the [end_session](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-phone-actions#phone-actions-hangup) response type.
+- How to transfer a call via a SIP Transfer using the [Connect To Agent Resolver](#transferring-the-call-via-the-sip-transfer-option-in-the-connect-to-agent-resolver)
+
+### Setting up your own assistant
+
+This example is configured to use an existing assistant set up for common use by anyone running this example. If you want to set up your own assistant, you'll need to perform the following steps.
+
+- Import the [actions.json](./action.json) file located in the repository for this example.
+- Optionally, to utilize the phone and SMS channel for 2-factor authentication, please create an [SMS integration](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-deploy-sms)
```

Comment:
rewrite: Create an SMS integration to use the phone and SMS channel for two-factor authentication (optional).

Reply:
```suggestion
2. Create an [SMS integration](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-deploy-sms) to use the phone and SMS channel for two-factor authentication (optional).
```